plugins/meta/portmap/main.go

```diff
@@ -89,12 +91,24 @@ func cmdAdd(args *skel.CmdArgs) error {
 		if err := forwardPorts(netConf, netConf.ContIPv4); err != nil {
 			return err
 		}
+		// Delete conntrack entries for UDP to avoid conntrack blackholing traffic
+		// due to stale connections. We do that after the iptables rules are set, so
+		// the new traffic uses them. Failures are informative only.
+		if err := deletePortmapStaleConnections(netConf.RuntimeConfig.PortMaps, unix.AF_INET); err != nil {
+			log.Printf("fail to delete conntrack connections for %s: %v", netConf.ContIPv4.IP, err)
```

Comment:
(keep this log line, though - it's useful)

wording: "failed to delete stale UDP conntrack entries for "